src/main/java/org/ihtsdo/authoringservices/service/SpellingListsService.java

```diff
@@ -33,12 +31,10 @@ public class SpellingListsService {
 
     public SpellingListsService(
             @Value("${aws.resources.enabled}") boolean awsResourceEnabled,
-            @Value("${aws.key}") String accessKey,
-			@Value("${aws.secretKey}") String secretKey,
 			@Value("${aws.s3.spell-check.bucket}") String bucket,
 			@Value("${aws.s3.spell-check.path}") String path) throws IOException {
 
-	    this.s3Client = new S3ClientImpl(new BasicAWSCredentials(accessKey, secretKey));
+	    this.s3Client = new S3ClientImpl(S3Client.builder().credentialsProvider(ProfileCredentialsProvider.create()).build());
```

Comment:
We don't use profile in our deployment so we should use DefaultAwsRegionProvider instead